content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,269 @@
+
+.. _https_certificate_iot:
+
+=======================
+HTTPS certificate (IoT)
+=======================
+
+What is HTTPS?
+==============
+
+*Hypertext Transfer Protocol Secure* (HTTPS) is the secure version of *Hypertext Transfer Protocol*
+(HTTP), which is the primary protocol used to send data back and forth between a web browser and a
+website. :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` is encrypted in order to increase the
+security of data transfer.
+
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` uses an encryption protocol to encrypt
+communications. The protocol is called *Transport Layer Security* (TLS), although formerly it was
+known as *Secure Sockets Layer* (SSL).
+
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` occurs based upon the transmission of :abbr:`TLS
+(Transport Layer Security)`/:abbr:`SSL (Secure Sockets Layer)` certificates, which verify that a
+particular provider is who they say they are.
+
+.. note::
+   In this documentation and throughout Odoo the term "HTTPS certificate" will be used to define the
+   fact that the SSL certificate is valid and allows a :abbr:`HTTPS (Hypertext Transfer Protocol
+   Secure)` connection.
+
+
+Why is it needed?
+=================
+
+In order to communicate with certain network devices (in particular for payment terminals), the
+usage of :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` is mandatory. If the :abbr:`HTTPS
+(Hypertext Transfer Protocol Secure)` certificate isn't valid, some devices won't be able to
+interact with the :abbr:`IoT (Internet of Things)` Box.
+
+
+How to obtain a Hypertext Transfer Protocol Secure (HTTPS) certificate
+======================================================================
+
+The generation of the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate is automatic.
+
+The :abbr:`IoT (Internet of Things)` Box will send a specific request to `<https://www.odoo.com>`__
+which will send back the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate if the
+:abbr:`IoT (Internet of Things)` box and database are eligible.
+
+Internet of Things (IoT) eligibility
+------------------------------------
+
+ #. The database should be a **production** instance.
+    The database instance should not be a copy, a duplicate, a staging or a development environment.
+ #. The Odoo subscription must:
+
+    - Have an :guilabel:`IoT Box Subscription` line.
+    - Be in progress.
+
+    If the subscription is linked to a `<https://www.odoo.com>`__ portal user check the information
+    on the portal subscription page.
+
+    .. figure:: https_certificate_iot/sub_example_in_progress.png
```

Comment:
Media file names should be hyphenated (no underscores), see: https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#use-single-underscore-suffixes-for-hyperlink-references

(This is repeated throughout the doc)

This image was also too wide, reduce width to the recommended 768px wide.